account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdatePasswordHandler.java

```diff
@@ -90,6 +103,18 @@ public APIGatewayProxyResponseEvent updatePasswordRequestHandler(
                                         objectMapper.readValue(
                                                 input.getBody(), UpdatePasswordRequest.class);
 
+                                Optional<ErrorResponse> passwordValidationError =
+                                        passwordValidator.validate(
+                                                updatePasswordRequest.getNewPassword());
+
+                                if (passwordValidationError.isPresent()) {
+                                    LOG.info(
+                                            "Error message: %s",
+                                            passwordValidationError.get().getMessage());
```

Comment:
I don't think this will log it how you think. Needs to use '{}' instead of '%s'

```suggestion
                                    LOG.info(
                                            "Error message: {}",
                                            passwordValidationError.get().getMessage());
```